includes/view/notice.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Notice output 
+ * 
+ * @package WSBC
+ */
+
+$adminurl = admin_url();
+$pluginpage = admin_url('plugins.php');
```

Comment:
use snake case for the name

```suggestion
$dashboard_url = admin_url();
$plugins_url   = admin_url('plugins.php');
```